CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.6.2 (Nov 7th, 2022)
+
+* Pinned SQLAlchemy `<=1.4.41` to avoid breaking changes (#520).
```

Comment:
Maybe worth pointing to the issue? https://github.com/encode/databases/issues/512